docs/end-user/component-dependency-parameter.md

```diff
@@ -2,4 +2,67 @@
 title:  Data Pass Between Components 
 ---
 
-WIP
\ No newline at end of file
+This section will introduce how to pass data between components.
+
+## How to use
+
+If we want to apply a WordPress server with data stored in a MySQL database, we need to pass the MySQL address to WordPress.
+
+Apply the following `Application`:
+
+```yaml
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: wordpress-with-mysql
+  namespace: default
+spec:
+  components:
+    - name: mysql
+      type: helm
+      outputs:
+        # the output is the mysql service address
+        - name: mysql-svc
+          exportKey: output.metadata.name + ".default.svc.cluster.local"
```

Comment:
Isn't this not a key but a value?

Reply:
I think this demo not work, please confirm @FogDong

Reply:
Talked to @FogDong . She will rectify it.

Reply:
The exportKey can be made up of key and value, the test has been passed locally.

Ref: https://github.com/oam-dev/kubevela/blob/master/pkg/controller/core.oam.dev/v1alpha2/application/generator_test.go#L103